examples/redis-upstash-session/sessions/0159/ca19a17d3092

```diff
@@ -0,0 +1 @@
+{"data":{"myStoredData":"some data"},"expires":"2022-01-07T15:06:48.773Z"}
```

Comment:
I think the `sessions` folder could be removed from the example, no? 🤔

Reply:
Yes it was a leftover. 10x